app.rb

```diff
@@ -2,4 +2,18 @@
 
 class Chitter < Sinatra::Base
   enable :sessions
+
```

Comment:
Hard to tell by these commits whether you tested first and then wrote the code or vice versa (I'm not a TDD advocate though so I don't mind :))

Reply:
I usually only push to Github once my tests are green, so I understand that can be a little confusing. I'll try to do commit more frequently to better show my approach. 😎